app/config/backdrop-demo/download.sh

```diff
@@ -20,8 +20,14 @@ pushd "$WEB_ROOT/web/modules" >> /dev/null
   git clone ${CACHE_DIR}/civicrm/civicrm-core.git      -b "$CIVI_VERSION"     civicrm
   git clone ${CACHE_DIR}/civicrm/civicrm-backdrop.git  -b "1.x-$CIVI_VERSION" civicrm/backdrop
   git clone ${CACHE_DIR}/civicrm/civicrm-packages.git  -b "$CIVI_VERSION"     civicrm/packages
-  git clone ${CACHE_DIR}/civicrm/api4.git              -b "master"            civicrm/ext/api4
-
+  case "$CIVI_VERSION" in
+    5.17|5.18)
+     git clone "${CACHE_DIR}/civicrm/api4.git"                                -b "4.5.2"             civicrm/ext/api4
+     ;;
+  *)
+     EXTCIVIVER=$( php -r '$x=simplexml_load_file("civicrm/xml/version.xml"); echo $x->version_no;' )
+     cv dl -b "@https://civicrm.org/extdir/ver=$EXTCIVIVER|cms=Drupal|status=|ready=/org.civicrm.api4.xml" --to="$WEB_ROOT/web/modules/civicrm/ext/api4" --dev
+  esac
```

Comment:
@seamuslee001 A few thoughts...

* I suppose we're not likely to remember to update this when/if there's a new tag of `4.5.x`.  There are git branches for previous versions (`4.4`, `4.3`) -- so I guess it makes sense to add one for `4.5`. (Done.)
* The `info.xml` for `api4@4.5` says that it requires `<ver>5.18</ver>`; and here were going for `4.5` on either `5.17|5.18`. This begs the question of whether `5.17` is actually compatible with `4.5`. I'll do a little test...
* Supposing that `4.5` is published... maybe the `cv dl -b` command is the only one we need? Don't need the `git clone`?

Reply:
* For purposes of merging `api4` into `civicrm-core` (https://github.com/civicrm/org.civicrm.api4/issues/198), we actually want to *not* download `api4` going forward. The basic rationale for checking out `api4@4.5` is (paradoxically) to allow `api_v3_AllTests` to continue passing on `5.17` and `5.18`.